### PR TITLE
added-hector-slam and fixed-frontier-exploration on melodic-devel branch

### PR DESCRIPTION
**Pull request by basavarajnavalgund**
## How to run hector_slam with husky

To run **gazebo with spawned robot**

```
cd ~/catkin_ws/
roslaunch husky_gazebo husky_empty_world.launch
```	

To run **hector_slam_demo**

```
cd ~/catkin_ws
roslaunch husky_navigation hector_slam_demo.launch 
```

## How to run frontier_exploration with husky

To run **gazebo with spawned robot**

```
cd ~/catkin_ws/
roslaunch husky_gazebo husky_empty_world.launch
```	

To install **frontier_exploration** package

```
cd ~/catkin_ws/src
git clone https://github.com/paulbovbel/frontier_exploration.git -b melodic-devel
cd ~/catkin_ws/
catkin_make --only-pkg-with-deps exploration_msgs
catkin_make -DCATKIN_WHITELIST_PACKAGES=""
catkin_make
source devel/setup.bash
```

To run **frontier_exploration_demo**

```
cd ~/catkin_ws
roslaunch husky_navigation frontier_exploration_demo.launch 
```

### CODE REVIEW

**Comment by tonybaltovski** [comment]
@basavarajnavalgund `frontier_exploration` was removed since it was not released for melodic.  It can't be re-added till it gets bloomed.

File: .vscode/c_cpp_properties.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (tonybaltovski) [inline .vscode/c_cpp_properties.json]: Remove this please.

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (tonybaltovski) [inline .vscode/settings.json]: Remove this please.

**Review by tonybaltovski — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 2 above)
Comment (basavarajnavalgund) [reply .vscode/settings.json]: Sure i'll do..

Comment (basavarajnavalgund) [reply .vscode/settings.json]: Done i've removed... Please check my latest commit a97b15e67647e9121b43a79a7da7a19bce8dcb03

File: .vscode/c_cpp_properties.json (hunk 1 above)
Comment (basavarajnavalgund) [reply .vscode/c_cpp_properties.json]: Done i've removed.. Please check my latest commit a97b15e67647e9121b43a79a7da7a19bce8dcb03

**Review by tonybaltovski — CHANGES_REQUESTED** [review]
This is missing a dependency on `hector_slam`  Again, frontier_exploration was removed since it was not released for melodic. It can't be re-added till it gets bloomed.

**Review by basavarajnavalgund — COMMENTED** [review]
> This is missing a dependency on `hector_slam` Again, frontier_exploration was removed since it was not released for melodic. It can't be re-added till it gets bloomed.

Ah okay when released for melodic we can add..
Added dependencies for hector_slam. Please check latest commit 15052639f9e54a421710f8c78423060f7fcb4be6

**Pull request closed without merging**